content/en/docs/contribute/style/style-guide.md

```diff
@@ -152,7 +152,7 @@ Run the process as a DaemonSet in the `kube-system` namespace. | Run the process
 {{< table caption = "Do and Don't - Use code style for Kubernetes command tool and component names" >}}
 Do | Don't
 :--| :-----
-The kubelet preserves node stability. | The `kubelet` preserves node stability.
+The `kubelet` preserves node stability. | The kubelet preserves node stability.
```

Comment:
If the style guide changes, do you plan to update the docs too?

Reply:
Right. But after discussing this change here, I'll propose to improve the guideline phrasing (instead of updating the style guide perse)